tests/sles4sap/qesapdeployment/cluster_add_repos.pm

```diff
@@ -0,0 +1,60 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Deployment steps for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use mmapi 'get_current_job_id';
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+sub run() {
+    my ($self, $run_args) = @_;
+    my $instance = $run_args->{my_instance};
+    record_info("$instance");
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    my $count = 0;
+
+    while (defined(my $maintrepo = shift @repos)) {
+        next if $maintrepo =~ /^\s*$/;
+        qesap_ansible_cmd(cmd => "zypper --no-gpg-checks ar -f -n TEST_$count $maintrepo TEST_$count",
+            provider => $prov);
+        $count++;
+    }
+
+    qesap_ansible_cmd(cmd => 'sudo zypper -n ref', provider => $prov);
+}
+
+sub delete_peering {
+    # destroy the network peering, if it was created
+    my $rg = qesap_get_az_resource_group();
+    my $vn = qesap_get_vnet($rg);
+    my $target_rg = get_required_var('QESAP_TARGET_RESOURCE_GROUP');
```

Comment:
what is this? the resource group of the IBS Mirror?

Reply:
yes!